docs/_static/css/igwnauthutils.css

```diff
@@ -0,0 +1,30 @@
+/* style sheet for IGWN Auth Utils docs */
```

Comment:
How is this stylesheet related to the present PR?

Reply:
This improves the documentation which has been augmented to include docs of the new requests interface. It can easily be moved to a new PR.

Reply:
Ok, that makes sense. I think it's fine to leave it in the same PR.